### ISSUE TITLE
Phonegap Support

### ISSUE THREAD

**Issue by dcousineau** [spazproject/Spaz-Enyo#60]
I did a test compile of Spaz Enyo using Phonegap straight into the IOS emulator. It worked almost completely out of box (some button taps weren't working on like the modal butons for some reason), but absolutely no changes made.

Obviously this is the lowest of priorities, but with such good functionality already with no effort, this is definitely a viable path to deploy Spaz to non-webos platforms... Of course assuming Enyo is licensed in such a manner...

**Comment by funkatron** [comment]
Yep. If you saw my blog you know I'm really pulling for HP not to fuck this up. I fully intend to release something for PhoneGap, at least on Android, and I'd like to do iOS as well (although I'm really leery of releasing it to the app store).

Would you mind giving us step-by-steps on getting this working in the iOS emulator?

**Comment by dcousineau** [comment]
Yeah, will post it on the wiki.

**Comment by dcousineau** [comment]
Creepy, you JUST posted on this topic didn't you? :P

**Comment by funkatron** [comment]
creepy

**Comment by dcousineau** [comment]
Added wiki page: https://github.com/funkatron/Spaz-Enyo/wiki/Phonegap%2C-iOS-and-XCode4
